src/scripts/main.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
-// write code here
+const addRowBtn = document.querySelector('.append-row');
+const addColumnBtn = document.querySelector('.append-column');
+const deleteRowBtn = document.querySelector('.remove-row ');
+const deleteColumnBtn = document.querySelector('.remove-column');
+
+const table = document.querySelector('.field');
+
+const maxCount = 10;
+const minCount = 2;
+let rowCount = table.rows.length;
+let columnCount = table.rows[0].cells.length;
+
+function updateBtnsStates() {
+  addRowBtn.disabled = rowCount >= maxCount;
+  deleteRowBtn.disabled = rowCount <= minCount;
+  addColumnBtn.disabled = columnCount >= maxCount;
+  deleteColumnBtn.disabled = columnCount <= minCount;
+}
+
+addRowBtn.addEventListener('click', () => {
+  const newRow = table.insertRow();
+
+  for (let i = 0; i < columnCount; i++) {
+    const newCell = newRow.insertCell();
+
+    newCell.textContent = '';
+  }
+
+  rowCount++;
+  updateBtnsStates();
+});
```

Comment:
This block of code is responsible for adding a row to the table. It seems that you are adding an empty row. If this is the expected behavior, then it's fine. However, if you want to add some default data or format to the cells, you should do it here.